Scripts/RemediationScripts/Remediate-DisableRemoteDebuggingForAppServices.ps1

```diff
@@ -416,27 +527,49 @@ function Disable-RemoteDebuggingForAppServices
             {
                 try
                 {
+                    Write-Host "Disabling Remote Debugging on the production slot..." -ForegroundColor $([Constants]::MessageType.Info)
+                    Write-Host $([Constants]::SingleDashLine)
                     $resource = Get-AzWebApp -ResourceGroupName $resourceGroupName -Name $resourceName
                     $resource.SiteConfig.RemoteDebuggingEnabled = $false
-
                     # Holding output of set command to avoid unnecessary logs.
                     $temp = $resource | Set-AzWebApp -ErrorAction SilentlyContinue
                     $isRemoteDebuggingDisabledOnProductionSlot = -not $(Get-AzWebApp -ResourceGroupName $resourceGroupName -Name $resourceName).SiteConfig.RemoteDebuggingEnabled
 
                     if ($isRemoteDebuggingDisabledOnProductionSlot)
                     {
-                        $isProductionSlotRemediated = $true
                         $appService.IsRemoteDebuggingDisabledOnProductionSlotPostRemediation = $true
+                        # $logResource = @{}
+                        # $logResource.Add("ResourceGroupName",($_.ResourceGroupName))
+                        # $logResource.Add("ResourceName",($_.ResourceName))
+                        # $logRemediatedResources += $logResource
+                        Write-Host "Successfully disabled remote debugging on the production slot." -ForegroundColor $([Constants]::MessageType.Update)
+                        Write-Host $([Constants]::SingleDashLine)
                     }
                     else
                     {
                         $appServicesSkipped += $appService
+                        $logResource = @{}
```

Comment:
This log resource code has been repeated many times, we could have used some common helper function instead.